pages/help.tsx

```diff
@@ -73,13 +74,11 @@ export default function Help() {
                   </Typography>
                 </CardContent>
                 <CardActions>
-                  <Button
-                    variant='contained'
-                    href={uiConfig.help?.documentationUrl}
-                    sx={{ margin: 'auto', mb: 1.5, width: 200 }}
-                  >
-                    View documentation
-                  </Button>
+                  <Link passHref href='/docs'>
+                    <Button variant='contained' sx={{ margin: 'auto', mb: 1.5, width: 200 }}>
```

Comment:
Really minor, but "margin" can just be "m"

Reply:
Done